### PR TITLE
add maximum units on a truck python3 / java

### PR DESCRIPTION
**Pull request by snowan**

### CODE REVIEW

**Comment by snowan** [comment]
@sunboman need help review python3?

File: 2021/python/maximum-units-on-truck.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      1 <= numberOfBoxesi, numberOfUnitsPerBoxi <= 1000
+      1 <= truckSize <= 106
+
+"""
+from typing import List
+def maximumUnits(boxTypes : List[List[int]], truckSize: int) -> int:
+    if not boxTypes or truckSize == 0:
+        return 0
+    res = 0
+    boxTyes.sort(key = lambda x: -x[1])
```
Comment (sunboman) [inline 2021/python/maximum-units-on-truck.py]: typo!

File: 2021/python/maximum-units-on-truck.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    res = 0
+    for b, unit in boxTypes:
+        take = truckSize if truckSize - b < 0 else b
+        res += take * unit
+        tuckSize -= b
```
Comment (sunboman) [inline 2021/python/maximum-units-on-truck.py]: typo2!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Review by sunboman — CHANGES_REQUESTED** [review]

File: 2021/python/maximum-units-on-truck.py (hunk 1 above)
Comment (snowan) [reply 2021/python/maximum-units-on-truck.py]: done

File: 2021/python/maximum-units-on-truck.py (hunk 2 above)
Comment (snowan) [reply 2021/python/maximum-units-on-truck.py]: fixed, booooo

**Pull request merged**